pkg/storegateway/indexheader/header.go

```diff
@@ -45,7 +46,7 @@ type Reader interface {
 
 	// LookupSymbol returns string based on given reference.
 	// Error is return if the symbol can't be found.
-	LookupSymbol(o uint32) (string, error)
+	LookupSymbol(ctx context.Context, o uint32) (string, error)
```

Comment:
Do we need the context on this one? I can't see where we need it upstream.

Reply:
It's used in [index.Decoder.LookupSymbol](https://github.com/grafana/mimir/blob/fce0b89/pkg/storegateway/bucket_index_reader.go#L62), but I see that we can circumvent it as the context isn't used in our code. I can make a revision to remove the context argument.

Reply:
Removed the context in latest revision, thanks for pointing this out.